ipaplatform/base/constants.py

```diff
@@ -8,15 +8,19 @@
 
 
 class BaseConstantsNamespace(object):
+    ANON_USER = 'WELLKNOWN/ANONYMOUS'
     DS_USER = 'dirsrv'
     DS_GROUP = 'dirsrv'
     HTTPD_USER = "apache"
+    GSSPROXY_USER = "root"
     IPA_DNS_PACKAGE_NAME = "freeipa-server-dns"
     KDCPROXY_USER = "kdcproxy"
     NAMED_USER = "named"
     NAMED_GROUP = "named"
     PKI_USER = 'pkiuser'
     PKI_GROUP = 'pkiuser'
+    IPAAPI_USER = 'ipaapi'
+    IPAAPI_GROUP = 'ipaapi'
```

Comment:
I thought it "may" be a platform specific thing, in case some distro has "opinions" on the user names. That said I can move it but I will keep ipaapi, I really like the palindrome and do nto want potential issues (bas scripts, user naming policies, and what not) with adding a dash in the name.